topiary/cli/script.py

```diff
@@ -54,12 +50,6 @@ def main(args_list=None):
     args = parse_args(args_list)
     print("Topiary commandline arguments:")
     print(args)
-    epitopes = predict_epitopes_from_args(args)
```

Comment:
Am I understanding this right, that `predict_epitopes_from_args` used to grab `gene_expression_dict` internally, but we also needed to grab `gene_expression_dict` separately in the next line to pass it through to `epitopes_to_dataframe`?

Why would that have been necessary if the epitopes were already filtered in `predict_epitopes_from_args`? And if they weren't filtered, how will `df = predict_epitopes_from_args(args)` work?